wazo_confd/plugins/endpoint_sip/schema.py

```diff
@@ -70,6 +81,18 @@ class EndpointSIPSchema(_BaseSIPSchema):
     line = fields.Nested('LineSchema', only=['id', 'links'], dump_only=True)
 
 
+class MergedEndpointSIPSchema(EndpointSIPSchema):
```

Comment:
I love that version :+1:

But you removed the option merging when you removed the post-dump

Reply:
thank you, fixed